lib/Parser/Parser.cpp

```diff
@@ -1136,11 +1211,12 @@ bool Parser::parseLine(std::string &ErrStr) {
               ErrStr = makeErrStr(TP, "expected ')' to complete data flow fact string");
```

Comment:
Just for reference. That's what I was mentioning in my previous comment for the parser unit test.